app/models/game.rb

```diff
@@ -5,4 +5,128 @@ class Game < ActiveRecord::Base
   belongs_to :white_user, class_name: 'User', foreign_key: 'white_uid'
   belongs_to :black_user, class_name: 'User', foreign_key: 'black_uid'
 
+  after_create :populate_board!
+  
+ 	def populate_board!
+ 		game = Game.last.id
+	
+		######create rooks######
+		0.step(7,7) do |x|
+			Piece.create!(
```

Comment:
something like: [This Gist](https://gist.github.com/cryogenx/d23c12245b4c33cb8faa#file-pawns-rb)